### PR TITLE
Changed Kilostation's Solars

### PR DESCRIPTION
**Pull request by EuSouAFazer**
## About The Pull Request

Changed Kilostation's solars to be more like Meta's. This will hopefully make wiring the solars more interesting and less monotonous as it's not just putting wire in a straight line down, in addition to each solar doing a different thing.

Medbay's solar now starts with a Space Carp that cannot destroy stuff.
Security's solar now doesn't have a solar computer, instead having an empty computer frame and the circuitboard nearby. There is also 2 glass nearby as well.
Engineering's solar now starts partially built, with a box with assembly and glass nearby.
Science's solars weren't changed.

:cl:
expansion: Kilostation Medbay's solar now starts with a Space Carp that cannot destroy stuff.
expansion: Kilostation Security's solar now doesn't have a solar computer, instead having an empty computer frame and the circuitboard nearby. There is also 2 glass nearby as well.
expansion: Kilostation Engineering's solar now starts partially built, with a box with assembly and glass nearby.
/:cl:

### CODE REVIEW

File: _maps/map_files/KiloStation/KiloStation.dmm [hunk 1]
```diff
@@ -43364,6 +43364,12 @@
 /obj/structure/cable,
 /turf/open/floor/iron/white,
 /area/security/prison)
+"ehM" = (
+/mob/living/simple_animal/hostile/carp{
```
Comment (ATH1909) [inline _maps/map_files/KiloStation/KiloStation.dmm:43368]: if you could give this carp a different name or some other way of indicating that it's different from a normal carp, that'd be nice.

**Comment by godspeedfunnyman** [comment]
👍 
but why add the walls back here
image

**Comment by CRITAWAKETS** [comment]
I would recommend adding 2 glass to the sec solars, since its far enough from engineering where having to get external material in order to get to the computer to work would be incredibly annoying.

**Comment by EuSouAFazer** [comment]
OK; thanks y'all for the feedback.
Sorry about the medbay wall thing; I'm still pretty new n such. Gonna use the map merger n such to fix that.

I'm also going to add some glass to engineering and rename the carp. DO NOT MERGE THE PULL REQUEST YET - I STILL HAVE TO DO THOSE CHANGES.

**Comment by godspeedfunnyman** [comment]
> OK; thanks y'all for the feedback.
> Sorry about the medbay wall thing; I'm still pretty new n such. Gonna use the map merger n such to fix that.
> 
> I'm also going to add some glass to engineering and rename the carp. DO NOT MERGE THE PULL REQUEST YET - I STILL HAVE TO DO THOSE CHANGES.

You can fix this by downloading this https://github.com/tgstation/tgstation/blob/master/_maps/map_files/tramstation/tramstation.dmm and redoing your changes

**Comment by Okand37** [comment]
epic

**Comment by EuSouAFazer** [comment]
Finally! All of the map merging problems has been fixed.

The carp has also been named Tuna, and 2 glass sheets has been added near the security solars so the engineers who forgot to bring glass don't have to go back and grab them.

File: _maps/map_files/KiloStation/KiloStation.dmm (hunk 1 above)
Comment (EuSouAFazer) [reply _maps/map_files/KiloStation/KiloStation.dmm:43368]: Did that'!

**Pull request merged**